### PR TITLE
Include .Release.Namespace in chart

### PR DESCRIPTION
**Pull request by dmyerscough**
## What this PR does / why we need it:

This PR includes .Release.Namespace that is required when doing Helm deployments from Spinnaker.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by Smirl** [comment]
/assign @gianrubio

@dmyerscough Could you please rename this to include the chart name in the title. Makes it easier to see what chart you are changing. For example:

*[stable/prometheus-node-exporter] Include .Release.Namespace in chart*

**Comment by dmyerscough** [comment]
@Smirl just updated to include your recommendations

**Comment by dmyerscough** [comment]
@Smirl @gianrubio any chance we can get this merged in?

**Comment by zanhsieh** [comment]
@dmyerscough 
Please bump chart to the latest. Thank you.

**Comment by dmyerscough** [comment]
@zanhsieh bumped version

**Comment by dmyerscough** [comment]
Any chance this can be merged?

**Comment by zanhsieh** [comment]
@maorfr @davidkarlsen 
Would you mind to help to LGTM this PR please? @gianrubio is the solely owner of this chart and he lost interest.

**Comment by dmyerscough** [comment]
Bump

**Comment by zanhsieh** [comment]
@vsliouniaev 
Would you mind to help with this PR please?

File: stable/prometheus-node-exporter/templates/daemonset.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: {{ template "prometheus-node-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (vsliouniaev) [inline stable/prometheus-node-exporter/templates/daemonset.yaml:5]: Please replace `{{ .Release.Namespace }}` with `{{ $.Release.Namespace }}` This is less error prone, especially when folks start copy-pasting it.

File: stable/prometheus-node-exporter/templates/psp-clusterrolebinding.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ kind: ClusterRoleBinding
 metadata:
   labels: {{ include "prometheus-node-exporter.labels" . | indent 4 }}
   name: psp-{{ template "prometheus-node-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (vsliouniaev) [inline stable/prometheus-node-exporter/templates/psp-clusterrolebinding.yaml:8]: This is not a namespaced resource

File: stable/prometheus-node-exporter/templates/psp-clusterrole.yaml [hunk 3]
```diff
@@ -5,6 +5,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   labels: {{ include "prometheus-node-exporter.labels" . | indent 4 }}
   name: psp-{{ template "prometheus-node-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (vsliouniaev) [inline stable/prometheus-node-exporter/templates/psp-clusterrole.yaml:8]: This is not a namespaced resource

**Review by vsliouniaev — CHANGES_REQUESTED** [review]
You have ticked 

>Title of the PR starts with chart name (e.g. [stable/chart])

but it does not.

**Comment by vsliouniaev** [comment]
I do not have rights to LGTM this chart

**Pull request closed without merging**